server/src/main/java/org/elasticsearch/action/support/RetryableAction.java

```diff
@@ -118,6 +118,10 @@ protected long calculateDelay(long previousDelay) {
         return Math.min(previousDelay * 2, Integer.MAX_VALUE);
     }
 
+    protected long minimumDelay() {
```

Comment:
Could you please add some unit indication? Like `minimumDelayMillis()`?

Reply:
100%